src/Internal/Builtin/MiniBillCodec.elm

```diff
@@ -0,0 +1,127 @@
+module Internal.Builtin.MiniBillCodec exposing (codeGen)
+
+import CodeGenerator exposing (CodeGenerator)
+import Elm.CodeGen as CG
+import ResolvedType
+import String.Extra
+import TypePattern exposing (TypePattern(..))
+
+
+val =
+    CG.fqVal [ "Codec" ]
+
+
+fn1 name arg =
+    CG.apply [ CG.fqVal [ "Codec" ] name, arg ]
+
+
+codeGen : CodeGenerator
+codeGen =
+    CodeGenerator.define
+        { id = "miniBill/elm-codec/Codec"
+        , dependency = "miniBill/elm-codec"
+        , typePattern = Typed [ "Codec" ] "Codec" [ Target ]
+        , makeName = \name -> String.Extra.decapitalize name ++ "Codec"
+        }
+        [ CodeGenerator.int (val "int")
+        , CodeGenerator.float (val "float")
+        , CodeGenerator.string (val "string")
+        , CodeGenerator.list (fn1 "list")
+        , CodeGenerator.maybe (fn1 "nullable")
```

Comment:
```suggestion
        , CodeGenerator.use "Codec.nullable"
```

This might fix the Maybe related issues. This is an annoying workaround, but I haven't quite figured out how to fix this issue properly.

Reply:
I tried that but it doesn't seem to work either. The thing is, I first used `Codec.maybe` which wasn't correct but ought to have worked for code gen but didn't for some reason. I'm confused what difference there could be that causes elm-serialize to correctly handle Maybe but not elm-codec.

Reply:
Ok I’ll try to find some time on Monday to look into it a bit more.

Reply:
OK, I fixed this, it turned out there was a stupid oversight in the code that was managing this stuff which just happened to work for the other use case because of different source order :(